src/routes.rs

```diff
@@ -175,14 +175,17 @@ pub mod data {
     }
     pub mod get {
         use crate::render::{RenderTemplate, Rendered, Renderer};
-        use crate::storage::{Data, Storer};
+        use crate::storage::Storer;
         use crate::token::TokenGenerator;
         use serde::{Deserialize, Serialize};
         use std::collections::HashMap;
         use warp::{Filter, Rejection, Reply};
         use warp_sessions::{
             self, CookieOptions, SameSiteCookieOption, Session, SessionStore, SessionWithStore,
         };
+        use crate::redis_client::FetchCache;
+
+        const PAGE_SIZE: u8 = 10; // TODO: this shouldn't be a constant and is already being pulled from config for storage&redis
```

Comment:
Just a reminder to get rid of it before merging

Reply:
done